samples/scenarios-monitoring/resource-health-alerts.bicep

```diff
@@ -0,0 +1,21 @@
+param activityLogAlertName string = uniqueString(resourceGroup().id)
+
+resource resourceHealthAlert 'Microsoft.Insights/activityLogAlerts@2020-10-01' = {
+  name: activityLogAlertName
+  location: 'global'
+  properties: {
+    condition: {
+      allOf: [
+        { 
+          field: 'category'
+          equals: 'ServiceHealth'
+        }
+      ]
+    }
+    scopes: [
+      subscription().id
+    ]
+    actions: {
```

Comment:
Does the empty property need to be there?

Reply:
It does, but what I've done is populate the property with an action group id to make it more comprehensive